### DISCUSSION TITLE
logo contribution

### DISCUSSION THREAD

**Discussion by reallinfo** [django-recaptcha/django-recaptcha#250] [Ideas]
Hi! I contribute to open source projects that do not have a logo. I am a logo designer. I have designed a functional logo to better represent the project. Also,I used the tones of the official django color. Please tell me what you think.

![recap](https://user-images.githubusercontent.com/36298335/57136734-66578d00-6db6-11e9-859d-aec47e3f71f0.jpg)

![recap2](https://user-images.githubusercontent.com/36298335/57136740-69eb1400-6db6-11e9-8e92-3da0f168fb56.png)

**Comment by markupguy** [comment]
Looks great @reallinfo !

**Comment by reallinfo** [comment]
thanks @markupguy

**Comment by rahulkp220** [comment]
Love these! Awesome.

**Comment by thibaudcolas** [comment]
Hey @reallinfo, this looks pretty cool :) If you can provide the first variation from your list as a SVG file, I think we’ll gladly use it for this project and credit you in the README.
